junit5/src/main/java/cz/xtf/junit5/extensions/OpenShiftRecorderService.java

```diff
@@ -475,6 +480,43 @@ protected void savePodLogs(ExtensionContext context, ResourcesFilterBuilder<Pod>
         }
     }
 
+    /**
+     * openshift log may be multiline, thus messages are separated with ESCAPE char
+     */
+    private static final String delimiter = "\u001B\\[.*?m";
+
+    protected void printPodLogs(ExtensionContext context, PrintStream ps) {
```

Comment:
I see most of the code is duplication of `OpenShiftRecorderService.savePodLogs(...)`. Could be added `PrintStream ps` as a new parameter to `OpenShiftRecorderService.savePodLogs(..., PrintStream ps)` and when not null added into it. Similarly with other methods.